mne/decoding/tests/test_base.py

```diff
@@ -241,35 +242,126 @@ def transform(self, X):
 
 @pytest.mark.parametrize("inverse", (True, False))
 @pytest.mark.parametrize(
-    "Scale, kwargs",
+    "clf",
     [
-        (Scaler, dict(info=None, scalings="mean")),
-        (_Noop, dict()),
+        make_pipeline(
```

Comment:
Okay one more change just to make our testing lives easier (prettier!)... you can do
```
pytest.param(make_pipeline(...), id="Scaler+SlidingEstimator),
pytest.param(make_pipeline(...), id="Noop+SlidingEstimator"),
pytest.param(make_pipeline(...), id="SlidingEstimator+nesting"),
```
or whatever you want to call them for a shorthand. It will make the `pytest` output more readable if it ever fails, and easier to subselect with `-k`.